deploy/helm/charts/values.yaml

```diff
@@ -53,6 +53,8 @@ zfsNode:
   priorityClass:
     create: true
     name: zfs-csi-node-critical
+  # allowed topologykeys for csi driver
+  allowedTopologyKeys: "kubernetes.io/hostname,"
```

Comment:
Can you make default to "All", same as operator yaml https://github.com/openebs/zfs-localpv/blob/develop/deploy/zfs-operator.yaml#L2350.